File: VBLib/Demos.vb
Imports System.IO

Public Class Demos
    Implements IDemos

    Public Function LoadFile() As List(Of String) Implements IDemos.LoadFile
        Dim outputs As List(Of String) = New List(Of String)
        Dim lines As List(Of String) = File.ReadAllLines("C:\temp\test.txt").ToList()
        For i As Integer = 0 To lines.Count - 1
            If i Mod 2 = 0 Then
                outputs.Add(lines(i))
            End If
        Next

        Return outputs
    End Function


    Public Sub PrintFullName(firstname As String, lastname As String) Implements IDemos.PrintFullName
        Console.WriteLine($"{firstname} {lastname}")
    End Sub
End Class

File: CSharpLib/Demos.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharpLib
{
    /// <summary>
    /// This is the CSharp class that will be used in the LanguageComparison project
    /// </summary>
    
    public class Demos
    {
        public List<string> LoadFile(){
            List<string> outputs = [];
            List<string> lines = File.ReadAllLines(@"C:\temp\test.txt").ToList();
            for(int i = 0; i < lines.Count; i++)
            {
                if(i % 2 == 0)
                {
                    outputs.Add(lines[i]);
                }
                
            }
            return outputs;
        }

        public void PrintFullName(string firstName, string lastName)
        {
            Console.WriteLine($"{firstName} {lastName}");
        }
    }
}
